docs/_assets/snippets.js

```diff
@@ -27,15 +28,23 @@ function MaterialComponentsSnippets() {
  */
 MaterialComponentsSnippets.prototype.init = function() {
```

Comment:
afaik this should be fine. This is just for the documentation website and for now will only ever be used inside `window.onload()`